source/includes/_providers.md

```diff
@@ -1 +1,309 @@
 # Providers
+
+Provider is a child class of [`User`](#users).
+Providers are your typical Providers.
+
+## Provider object
+
+Attributes | Type | Description
+---------- | ---- | -------
+max_travel_time | `float` | The maximum time of travel. _default 1.0_
+max_travel_meters | `float` | The maximum range of travel _default 10000.0_
+default_position | `geoPoint` | The Provider's position by default
+default_address | `address` | The Provider's address by default. _default lambda: Address_
+schedule | `schedule` | The Provider's schedule. _default lambda: Schedule_
+orders | `list` | List of [`Orders`](#orders)
+products | `list` | List of [`Products`](#products) of Provider. _default lambda: []_
+available_on_bank_holidays | `boolean` | _default `false`_
+country | `string` | The Provider's country. _default `'NOR'`_
+roles | `list` | Define the rolls that a Provider can have. _default ['provider']_
+
+## Retrieve an Product
+
+``` http 
+GET /providers HTTP/1.1 
+Content-Type: application/json 
+Authorization: Bearer <jwt> 
+X-Share-Api-Key: <shareactor-api-key> 
+Host: api.shareactor.io 
+``` 
+
+``` http
+HTTP/1.1 200 OK
+Content-Type: application/json
+
+{  
+   "first_name":"Mari",
+   "email":"johansenemil@gmail.com",
```

Comment:
let's not have this sort of data here ;). have a look at how we do with the users